hadoop-jobs/src/main/java/com/linkedin/hadoop/jobs/HdfsWaitJob.java

```diff
@@ -73,15 +73,19 @@ public void run() throws Exception {
     boolean failOnTimeout = Boolean.valueOf(_properties.getProperty("forceJobToFail"));
     boolean folderFound = false;
 
+    log.info("STATUS: Job started. Checking the directory at " + dirPath + " for fresh folders with a sleep interval of " + _properties.getProperty("sleepInterval", "1M"));
+
     while (System.currentTimeMillis() < endTime && !folderFound) {
       folderFound = checkDirectory(dirPath, freshness);
       if (!folderFound) {
+        log.info("STATUS: No fresh folders found during latest polling. Now sleeping for " + _properties.getProperty("sleepInterval", "1M") + " before polling again.");
+        log.info("STATUS: There are " + TimeUnit.MILLISECONDS.toMinutes(sleepTime) + " minutes remaining before a forced timeout.");
```

Comment:
Hmm, this doesn't look right - this is the sleepInterval, not the remaining time before a timeout